samples/nrf9160/fmfu_smp_svr/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _fmfu_smp_svr_sample:
+
+Full modem update over SMP Server Sample
+########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Full modem firmware update sample implements a Simple Management Protocol (SMP) server.
+SMP is a basic transfer encoding for use with the MCUmgr management protocol.
+For more information about MCUmgr and SMP, please see :ref:`device_mgmt`.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns
+
+.. include:: /includes/spm.txt
+
+Overview
+********
+
+The sample first deinitializes the :ref:`nrfxlib:nrf_modem`.
+Next, it registers a stat command and firmware upload and get hash commands to
+MCUMgr.
+
+The sample then goes into and idle loop waiting for communication over the serial line.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/fmfu_smp_svr`
```

Comment:
Should we discuss the UART overlay file here? How to use it, why it should be used, and how it will impact the arguments to the script.